docs/getting-started-guides/coreos/index.md

```diff
@@ -16,6 +16,10 @@ Guide and CLI tool for setting up a multi-node cluster on AWS. CloudFormation is
 
 <hr/>
 
+[**Bare Metal Multi-Node**](https://coreos.com/kubernetes/docs/latest/kubernetes-on-baremetal.html)
```

Comment:
Given the context below, do you want to just link straight to https://coreos.com/kubernetes/docs/latest/kubernetes-on-baremetal.html#automated-provisioning ?